dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.connectors-2.0.internal.feature

```diff
@@ -1,5 +1,5 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=io.openliberty.jakarta.internal.connectors-2.0
```

Comment:
...and then if this feature is an internal implementation feature (not API), I would drop `jakarta` from the name... `io.openliberty.connectors-2.0.internal`.